src/openfga.ne

```diff
@@ -230,12 +236,16 @@ _no_relations   -> "none" (_newline):*
 _naming         -> (("$"):? ( [a-z] | [A-Z] | [0-9] |  "_" |  "-" ):+) {%
     data => data.flat(3).join('').trim()
 %}
+_allowed_naming         -> (("$"):? ( [a-z] | [A-Z] | [0-9] |  "_" |  "-" | "#" ):+ (":*"):?) {%
```

Comment:
`$` should be only allowed on relations, not types. But let's have that conversation elsewhere. Not related to this PR.

Reply:
Let's track it via https://github.com/openfga/syntax-transformer/issues/96